config/settings.yml

```diff
@@ -19,3 +19,11 @@ marc_fixture_seeds:
     - Cornell
     - Princeton
     - Stanford
+
+# refer to https://icons.getbootstrap.com/ for icon class names
+metadata_status_icons:
```

Comment:
I'm wondering why this is in settings? It doesn't seem like something that we're likely to configure differently per environment. Maybe a helper method would work?

in `ApplicationHelper`:
```ruby
def metadata_status_icons(status)
  case status
  when :deletes then 'file-earmark-excel-fill'
  when :invalid then 'exclamation-circle-fill'
  # etc...
end
```